src/backend/distributed/commands/trigger.c

```diff
@@ -108,6 +122,29 @@ GetExplicitTriggerCommandList(Oid relationId)
 }
 
 
+/*
+ * GetDisableTriggerCommand returns the DDL command to disable given trigger.
+ */
+static char *
+GetDisableTriggerCommand(Oid triggerId)
+{
+	bool missingOk = false;
+	HeapTuple trigTup = GetTriggerTupleById(triggerId, missingOk);
+	Form_pg_trigger trigForm = (Form_pg_trigger) GETSTRUCT(trigTup);
+
```

Comment:
We can add an assert here
`Assert (triggerForm->tgenabled == TRIGGER_DISABLED)`

Reply:
Indeed I'd like to have this function not bound to enabled-triggers since this is only supposed to generate a command to disable given trigger idempotently, regardless of the fact that it's enabled or not.